include/flox/resolver/descriptor.hh

```diff
@@ -106,26 +106,23 @@ public:
 /* -------------------------------------------------------------------------- */
 
 /**
- * @fn void from_json( const nlohmann::json        & jfrom
- *                   ,       ManifestDescriptorRaw & desc
- *                   )
- * @brief Convert a JSON object to an @a flox::InputPreferences.
- *
- * @fn void to_json( nlohmann::json & jto, const ManifestDescriptorRaw & desc )
- * @brief Convert an @a flox::resolver::ManifestDescriptorRaw to a JSON object.
+ * @brief Convert a JSON object to an @a flox::ManifestDescriptorRaw. */
+void
+from_json( const nlohmann::json & jfrom, ManifestDescriptorRaw & descriptor );
+/** @brief Convert an @a flox::resolver::ManifestDescriptorRaw to a JSON
+ * object.
```

Comment:
```suggestion
/**
 * @brief Convert an @a flox::resolver::ManifestDescriptorRaw to a
 *              JSON object.
```

The brief messages have to be indented otherwise Doxygen will recognize them as the beginning of the "detailed documentation" block.

Also the `/** @... */` syntax is only accepted if the doc fits on one line. If you have a line break it requires `/**` on its own line.